server/node/drop_domain.go

```diff
@@ -79,8 +79,12 @@ func (c *DropDomain) RowIter(ctx *sql.Context, r sql.Row) (sql.RowIter, error) {
 	if err != nil {
```

Comment:
(This comment should be on the line above, thanks for getting $15B from microsoft and not addressing this feature request for 6 years and counting GitHub)

It's very obvious in this context that we should be getting the session then fetching the current working set from there, not using the context as indirection. But that's for another day.

Reply:
That's fair. Something to do when we make the `DoltgresSession`.